### PR TITLE
feat(contexts): Amazon Fire TV logo

### PR DESCRIPTION
**Pull request by bruno-garcia**
Don't recall how we add this, but opening this to get some input.

Just realized the device id changes with version: AFTT, AFTN, AFTS, AFTA, AFTVA (alexa) 

Perhaps a better approach here would be to take any device starting with `AFT` and rename it to `amazon`, name the class `.amazon` and use just an Amazon logo

image

### CODE REVIEW

**Comment by doralchan** [comment]
I'll svg-ize this and send it to you @bruno-garcia

**Comment by matejminar** [comment]
Sounds good, I believe this logic will need to be in:
https://github.com/getsentry/sentry/blob/f989583a8b0ef525bc15f53e1b8455ec46e76ae1/src/sentry/static/sentry/app/components/events/contextSummary/generateClassName.tsx

If you DMme fire tv event, I can test the payload locally and do the changes 👍

File: src/sentry/static/sentry/less/group-detail.less [hunk 1]
```diff
@@ -522,6 +522,10 @@
       background-image: url(../images/logos/context/ubuntu.png);
     }
 
+    &.amazon .context-item-icon {
```
Comment (bruno-garcia) [inline src/sentry/static/sentry/less/group-detail.less:525]: Trick now is to get the "known Amazon device ids" renamed to "amazon" to match this:

```suggestion
    &.amazon .context-item-icon {
```

**Comment by bruno-garcia** [comment]
Nice! Thanks @matejminar

**Review by matejminar — APPROVED** [review]

**Comment by bruno-garcia** [comment]
Thanks @matejminar and @doralchan

**Comment by bruno-garcia** [comment]
Forgot to attach an image of what this looked like:

image

**Pull request merged**